EIPS/eip-FOCIL.md

```diff
@@ -26,10 +26,82 @@ FOCIL is a simple committee-based design improving upon previous IL mechanisms o
 
 ## Specification
 
+### Constants
+
+| Name | Value |
+| - | - |
+| `MAX_TRANSACTIONS_PER_INCLUSION_LIST` | `2**4 = 16` |
+| `MAX_GAS_PER_INCLUSION_LIST` | `2**21` |
+| `MAX_TRANSACTIONS_PER_INCLUSION_LIST_AGGREGATE` | `2 * MAX_TRANSACTIONS_PER_INCLUSION_LIST = 32` |
+| `MAX_GAS_PER_INCLUSION_LIST_AGGREGATE` | `2 * MAX_GAS_PER_INCLUSION_LIST = 2**22` |
+| `IL_COMMITTEE_SIZE` | `256` |
+
+### Reference Objects
+
+```
+class InclusionListEntry(Container):
+    from_address: ExecutionAddress
+    gas_limit: uint64
+```
+
+```
+class InclusionList(Container)
+    slot: Slot
+    validator_index: ValidatorIndex
+    entries: List[InclusionListEntry, MAX_TRANSACTIONS_PER_INCLUSION_LIST]
+```
+
+```
+class InclusionListAggregatedEntry(Container):
+    from_address: ExecutionAddress
+    gas_limit: uint64
+    bitlist: List[byte, IL_COMMITTEE_SIZE]
+```
+
+```
+class InclusionListAggregated(Container):
+    slot: Slot
+    proposer_index: ValidatorIndex
+    message: List[InclusionListAggregatedEntry, MAX_TRANSACTIONS_PER_INCLUSION_LIST_AGGREGATE]
+```
+
 ### Consensus layer
 
 #### High-level overview
 
+TBD (Same as timeline)
+
+#### Specific changes
+
+**Beacon chain state transition spec:**
+
+- ***New** `inclusion_list` object:* Introduce a new `inclusion_list` for the IL committee to submit and nodes to process.
+- ***Modified** `BeaconBlockBody`:* Modified to include the aggregated inclusion list for that slot.
```

Comment:
to include the inclusion list aggregate

Reply:
we probably want to add a fork choice section as well to mention that a validator will not vote for a block that fails to satisfy eval

Reply:
What do you think is an ideal place to mention this? The `Eval` function is defined in another section below so I am not sure if it makes sense to mention this here. Let me know.

Reply:
Added in https://github.com/soispoke/EIPs/pull/6/commits/e3e433b7c832e6cd028837dd651fab92871927e0, please review. Thanks!